obspy/io/nordic/core.py

```diff
@@ -924,41 +1221,61 @@ def _write_nordic(event, filename, userid='OBSP', evtype='L', outdir='.',
         sfile = open(sfile_path, 'w')
     else:
         sfile = string_io
-    sfile.write(' ' + str(evtime.year) + ' ' +
-                str(evtime.month).rjust(2) +
-                str(evtime.day).rjust(2) + ' ' +
-                str(evtime.hour).rjust(2) +
-                str(evtime.minute).rjust(2) + ' ' +
-                str(evtime.second).rjust(2) + '.' +
-                str(evtime.microsecond).ljust(1)[0:1] + ' ' +
-                evtype.ljust(2) + lat.rjust(7) + ' ' + lon.rjust(7) +
-                depth.rjust(5) + agency.rjust(5) + ksta.rjust(3) +
-                timerms.rjust(4) +
-                conv_mags[0]['mag'].rjust(4) + conv_mags[0]['type'].rjust(1) +
-                conv_mags[0]['agency'][0:3].rjust(3) +
-                conv_mags[1]['mag'].rjust(4) + conv_mags[1]['type'].rjust(1) +
-                conv_mags[1]['agency'][0:3].rjust(3) +
-                conv_mags[2]['mag'].rjust(4) + conv_mags[2]['type'].rjust(1) +
-                conv_mags[2]['agency'][0:3].rjust(3) + '1' + '\n')
-    # Write line 2 of s-file
-    sfile.write(' ACTION:ARG ' + str(datetime.datetime.now().year)[2:4] + '-' +
-                str(datetime.datetime.now().month).zfill(2) + '-' +
-                str(datetime.datetime.now().day).zfill(2) + ' ' +
-                str(datetime.datetime.now().hour).zfill(2) + ':' +
-                str(datetime.datetime.now().minute).zfill(2) + ' OP:' +
-                userid.ljust(4) + ' STATUS:' + 'ID:'.rjust(18) +
-                str(evtime.year) +
-                str(evtime.month).zfill(2) +
-                str(evtime.day).zfill(2) +
-                str(evtime.hour).zfill(2) +
-                str(evtime.minute).zfill(2) +
-                str(evtime.second).zfill(2) +
-                'I'.rjust(6) + '\n')
-    # Write line 3 of s-file
+    # Write a second line if more than three magnitudes...
+    line_one_begin = (
+        ' ' + str(evtime.year) + ' ' + str(evtime.month).rjust(2) +
+        str(evtime.day).rjust(2) + ' ' + str(evtime.hour).rjust(2) +
+        str(evtime.minute).rjust(2) + ' ' + str(evtime.second).rjust(2) + '.' +
+        str(evtime.microsecond).ljust(1)[0:1] + ' ' + evtype.ljust(2) +
+        lat.rjust(7) + ' ' + lon.rjust(7) + depth.rjust(5) + agency.rjust(5) +
+        ksta.rjust(3) + timerms.rjust(4))
```

Comment:
This is for one really pretty slow as each `+` generates an intermediate string object. Could this not just be done with the `.format()` method of string objects?

Reply:
It definitely could and should! Will do.